README.md

```diff
@@ -1,20 +1,46 @@
-# Catone, the OpenCore config.plist censor
+![macOS](https://img.shields.io/badge/Supported_OC_build:-≥0.8.2-white.svg)
 
-The following script was created to censor sensitive fields in OpenCore's config.plist
+# OC Anonymizer – remove sensitive Data from your config.plist
 
-The reason why I created it is that many EFI I find online contain sensitive fields such as SMBIOS data.
-With this script you won't have any prob at all
+## About
+Python Script for removing sensitive Data from OpenCore's `config.plist` before sharing your Config/EFI folder online. It also changes some settings to default, such as: ScanPolicy, Custom Entries, LauncherOptions, etc. See details below. Basically it removes all those settings specific to your system and user preferences which may not be desireble to have on a different machine.
 
-# Usage
+## Features
 
-```bash
-$> python3 oc_anonymizer.py PATH_TO_CONFIG.plist
-```
+Changes the following Settings/Parameters in the **config.plist**:
 
-It'll create `censored_config.plist` with the stripped sensitive data
+- **SMBIOS** data:
+	- Deletes `PlatformInfo/Generic/MLB`
+	- Deletes `PlatformInfo/Generic/ROM`
+	- Deletes `PlatformInfo/Generic/SystemSerialNumber`
+	- Deletes `PlatformInfo/Generic/SystemUUID`
+- **Security Settings**:
+	- `Misc/Security/ApECID` = `0`
+	- `Misc/Security/ScanPolicy` = `0`
+	- `Misc/Security/SecureBootModel` = `Disabled` &rarr; Disables Apple Secure Boot hardware model to avoid issues during Installation. Re-enable in Post-Install so System Updates work when using an SMBIOS of a Mac model with a T2 Security Chip.
```

Comment:
I'm not sure `Disabled` `SecureBootModel` wouldn't cause issues during installation. Maybe adding a side note such as `if using a T2 SMBIOS Model set it to Default` or eventually adding a matching logic such as an array of T2 supported Macs so it doesn't break stuff

Reply:
Trust me: Installation issues usually occur when SecureBootModel is enabled and the user has some hardware combinations which conflict with it. Like using a GeForce Card in macOS Monterey with patched in drivers etc.

Reply:
Alright. Marking this as resolved